src/index.ts

```diff
@@ -33,20 +33,6 @@ const convertSolidFillStyle = (solidFillStyle: number): SolidFillPattern => {
   }
 };
 
-const convertInfillStyle = (infillStyle: number): InfillPattern => {
```

Comment:
Similar to the API, we should keep this fallback mapping in case `style.slic3rInfillStyle` has never been defined

Reply:
Actually never mind, I see that this lib's consumer always applies a default based on the existing `infillStyle` so no change needed here